Testscripts/Linux/dpdkPmdsBasicCheck.sh

```diff
@@ -59,15 +59,15 @@ function checkCmdExitStatus ()
 		SetTestStateAborted
 		exit $exit_status
 	else
-		echo "$cmd: SUCCESS" 
+		echo "$cmd: SUCCESS"
 	fi
 }
 
 runTestPmd()
 {
 	pmd=$1
 	LogMsg "*********INFO: Starting TestPmd test execution with ${pmd} PMD*********"
-	whitelist="-w ${bus_info}"
+	whitelist="-a ${bus_info}"
```

Comment:
how about use old dpdk?

Reply:
Let me confirm that with Luca. Currently, it is not there for 19.11.5, it can be there for 19.11.6 releasing in a month or so. 
We can add a check for DPDK version.

Reply:
fixed